app/code/Magento/ImportExport/Block/Adminhtml/Export/Filter.php

```diff
@@ -7,6 +7,11 @@
 
 use Magento\Eav\Model\Entity\Attribute;
 use Magento\Catalog\Api\Data\ProductAttributeInterface;
+use Magento\Catalog\Model\Product as CatalogProduct;
+use Magento\Framework\App\ObjectManager;
+use Magento\ImportExport\Model\Export\ConfigInterface;
+use Magento\CatalogImportExport\Model\Export\Product\Type\Factory as TypeFactory;
```

Comment:
Please do not use CatalogImportExport in the ImportExport module, this dependency should be eliminated

Reply:
@sivaschenko 
i have eleminated CatalogImportExport dependencies.
Please review.
Thanks.